examples/django-deployment/Makefile

```diff
@@ -22,10 +22,39 @@ test-and-coverage:
 .PHONY: compile-packages
 compile-packages:
 	PIPENV_VERBOSITY=-1                                \
-	mkdir -p ./artifacts && 									 \
+	mkdir -p ./artifacts && 									 				 \
 	cd ../../connectors/django/	&&  	 		        	   \
 	pipenv run python setup.py sdist bdist_wheel &&    \
 	cp -r dist/* ../../examples/django-deployment/artifacts/
-	cd ../../fma-core	&&  	          							 \
+	cd ../../fma-core	&&  	          							 	 \
 	pipenv run python setup.py sdist bdist_wheel &&    \
 	cp -r dist/* ../examples/django-deployment/artifacts/
+
+
+.PHONY: complete-install
+complete-install:
```

Comment:
could we rename this to just `install`? when I read `complete-install`, I infer that there are commands that are either prior to this one in sequence and/or other commands that are sub-comonents of this command that could be run on their own for the installation process

Reply:
maybe we hold off now, but want to at least raise this as something we can think about to make the intuition a bit cleaner here